src/jit/emitxarch.cpp

```diff
@@ -107,7 +108,7 @@ bool emitter::IsDstSrcSrcAVXInstruction(instruction ins)
 {
     return IsAVXInstruction(ins) &&
            (ins == INS_movlpd || ins == INS_movlps || ins == INS_movhpd || ins == INS_movhps || ins == INS_movss ||
-            ins == INS_movlhps || ins == INS_sqrtss || ins == INS_sqrtsd || ins == INS_cvtss2sd || ins == INS_cvtsd2ss);
```

Comment:
`INS_movlhps` was incorrectly listed as `DstSrcSrc`, which will cause it to generate a different result on AVX vs non-AVX machines.